pyproject.toml

```diff
@@ -17,6 +17,7 @@ typing-extensions = "*"
 pdbpp = "*"
 flake8 = "*"
 black = "*"
+parameterized = "*"
```

Comment:
It also needs to be addet to `test.yml`.

Reply:
Added parameterized to test.yml